packages/apps/graph-client/src/pages/index.tsx

```diff
@@ -51,13 +51,20 @@ const Home: React.FC = () => {
     }
   };
 
-  const searchTypePlaceholders: Record<string, string> = {
+  const searchTypeLabels: Record<string, string> = {
     'request-key': 'Request Key',
     account: 'Account',
     event: 'Event Name',
     block: 'Block Hash',
   };
 
+  const searchTypePlaceholders: Record<string, string> = {
+    'request-key': '',
+    account: 'k:1234...',
+    event: 'coin.TRANSFER',
+    block: '',
+  };
+
```

Comment:
In the cases of Request Key and Block is it better to leave an empty placeholder or to still have something like Enter block hash?
I think for the block especially it would be beneficial to have a placeholder since it's not self-explanatory like Request Key.

```suggestion
  const searchTypePlaceholders: Record<string, string> = {
    'request-key': 'Request Key',
    account: 'k:1234...',
    event: 'coin.TRANSFER',
    block: 'Block Hash',
  };

```

Reply:
Fixed!